pkgs/shells/bash/fzf-obc/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "fzf-obc";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "rockandska";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-KIAlDpt1Udl+RLp3728utgQ9FCjZz/OyoG92MOJmgPI=";
+  };
+
+  outputs = [ "out" ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/opt/${pname}/{bin,lib/${pname},plugins/{kill,gradle}}
+    install -m644 bin/* $out/opt/${pname}/bin
+    install -m644 lib/${pname}/* $out/opt/${pname}/lib/${pname}
+    install -m644 plugins/kill/* $out/opt/${pname}/plugins/kill
+    install -m644 plugins/gradle/* $out/opt/${pname}/plugins/gradle
+  '';
+
+  meta = with lib; {
+    homepage = "https://fzf-obc.readthedocs.io";
+    description = "Completion script adding fzf over all know bash completion functions";
+    license = "unknown";
```

Comment:
```suggestion
    license = licenses.unfree;
```

Reply:
Hey @SuperSandro2000, where's the guideline saying that for nixpkgs' packages, a missing license implies defaulting to unfree?